me.orszulak.keyring.json

```diff
@@ -0,0 +1,74 @@
+{
+	"app-id": "me.orszulak.keyring",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "41",
```

Comment:
does it really need the gnome platform? 

Gtk3 is standard in the freedesktop-sdk